src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/InputSeparatorWrapForTestComma2.java

```diff
@@ -0,0 +1,28 @@
+/*
+SeparatorWrap
+tokens = COMMA
+option = nl
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.separatorwrap;
+
+public class InputSeparatorWrapForTestComma2 {
```

Comment:
Please name test and input like `InputSeparatorWrapForTestTrailingWhitespace`

Reply:
Done